### PR TITLE
Weeding swagger-related code

### PR DESCRIPTION
**Pull request by Porges**
Tidying up some Swagger code while I look at the nondeterminism bug.

No functionality changes; stop threading some `spec.Swagger`s everywhere when we don’t need to.

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -403,7 +404,6 @@ tasks:
       - azure-crossplane.yaml
     cmds:
       - ./bin/{{.GENERATOR_APP}} gen-types azure-crossplane.yaml
-
```
Comment (matthchr) [inline Taskfile.yml:406]: Did you intentionally delete this line?

File: hack/generator/pkg/jsonast/swagger_type_extractor.go [hunk 2]
```diff
@@ -65,15 +68,15 @@ func (extractor *SwaggerTypeExtractor) ExtractTypes(
 			continue
 		}
 
-		resourceSchema := extractor.findARMResourceSchema(filePath, swagger, *put)
+		resourceSchema := extractor.findARMResourceSchema(filePath, *put)
 		if resourceSchema == nil {
-			//klog.Warningf("No ARM schema found for %s in %q", rawOperationPath, filePath)
+			// klog.Warningf("No ARM schema found for %s in %q", rawOperationPath, filePath)
```
Comment (matthchr) [inline hack/generator/pkg/jsonast/swagger_type_extractor.go:73]: Should this warning be commented out, or no?

**Review by theunrepentantgeek — COMMENTED** [review]
LGTM

gif

File: hack/generator/pkg/jsonast/swagger_type_extractor.go (hunk 2 above)
Comment (Porges) [reply hack/generator/pkg/jsonast/swagger_type_extractor.go:73]: Will address in a future PR; change here was formatter running

**Pull request merged**